### PR TITLE
[pytorch][eia][build]Handling Pytorch 1.5.1 eia vulnerabilities

### PR DESCRIPTION
**Pull request by jinpengqi**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Description
CVE fix for Pytorch 1.5.1 eia DLC
### Tests run
**NOTE: If you are creating a PR for a new framework version, please ensure success of the standard, rc, and efa sagemaker remote tests by updating the dlc_developer_config.toml file:**

- [ ] Revision A: `sagemaker_remote_tests = "standard"`
- [ ] Revision B: `sagemaker_remote_tests = "rc"`
- [ ] Revision C: `sagemaker_remote_tests = "efa"`

**Additionally, please run the sagemaker local tests in at least one revision:**
- [ ] `sagemaker_local_tests = true`

### DLC image/dockerfile
pytorch/inference/docker/1.5.1/py3/Dockerfile.eia

## Label Checklist
- [ ] I have added the project label for this PR (*<project_name>* or "Improvement")

## PR Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron/graviton] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [ ] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [x] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [x] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [x] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON/GRAVITON Testing Checklist
* When creating a PR:
- [x] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true`, `neuron_mode = true` or `graviton_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: dlc_developer_config.toml [hunk 1]
```diff
@@ -23,7 +23,7 @@ benchmark_mode = false
 [build]
 # Frameworks for which you want to disable both builds and tests
 # available frameworks - ["autogluon", "huggingface_tensorflow", "huggingface_pytorch", "huggingface_tensorflow_trcomp", "huggingface_pytorch_trcomp", "tensorflow", "mxnet", "pytorch"]
-skip_frameworks = []
+skip_frameworks = ["tensorflow", "mxnet"]
```
Comment (RadhikaB-97) [inline dlc_developer_config.toml:26]: Please add all the frameworks except "pytorch".

**Comment by RadhikaB-97** [comment]
Enable the sagemaker tests by setting `sagemaker_local_tests` to `true` and `sagemaker_remote_tests` to `standard`

**Comment by RadhikaB-97** [comment]
Set `dlc-pr-pytorch-eia` as` pytorch/buildspec-1-5-eia.yml` in dlc_developer_config.toml file

File: pytorch/buildspec-1-5-eia.yml [hunk 2]
```diff
@@ -29,9 +29,9 @@ images:
     build: &PYTORCH_CPU_INFERENCE_PY3 false
     image_size_baseline: 6318
     device_type: &DEVICE_TYPE cpu
-    python_version: &DOCKER_PYTHON_VERSION py3
-    tag_python_version: &TAG_PYTHON_VERSION py36
-    os_version: &OS_VERSION ubuntu16.04
+    python_version: &DOCKER_PYTHON_VERSION py38
```
Comment (arjkesh) [inline pytorch/buildspec-1-5-eia.yml:32]: The issue with the standard labels test appears to be arising from the fact that "device_type" is set to "cpu" instead of eia. Can we change device_type to "eia", and then in "tag" replace *DEVICE_TYPE with "cpu"

File: pytorch/buildspec-1-5-eia.yml [hunk 3]
```diff
@@ -29,9 +29,9 @@ images:
...
+    tag_python_version: &TAG_PYTHON_VERSION py38
```
Comment (arjkesh) [inline pytorch/buildspec-1-5-eia.yml:33]: question - why are we editing buildspec-1-5-eia.yml instead of buildspec-eia.yml?

File: pytorch/inference/docker/1.5.1/py38/Dockerfile.eia [hunk 4]
```diff
@@ -0,0 +1,137 @@
+FROM ubuntu:20.04
+LABEL maintainer="Amazon AI"
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL dlc_major_version="2"
```
Comment (arjkesh) [inline pytorch/inference/docker/1.5.1/py38/Dockerfile.eia:4]: It looks like a new python version constitutes as a new version stream. As a result, we can change this back to dlc_major_version="1"

File: pytorch/buildspec-1-5-eia.yml (hunk 2 above)
Comment (jinpengqi) [reply pytorch/buildspec-1-5-eia.yml]: Modified

File: pytorch/inference/docker/1.5.1/py38/Dockerfile.eia (hunk 4 above)
Comment (jinpengqi) [reply pytorch/inference/docker/1.5.1/py38/Dockerfile.eia]: Modified it back to "1"
